### ISSUE TITLE
Reader view available for Twitter, fails to load content

### ISSUE THREAD

**Issue by pdehaan** [mozilla/readability#141]
### Steps to reproduce:
1. Go to any Twitter acct, like https://twitter.com/realandreasgal
2. Click the "Enter Reader View" button up in the address bar.

### Actual results:
> #### _Failed to load article from page_

### Expected results:
No reader view available for twitter.com.

---

![failed_to_load_article_from_page_and_new_issue_ _mozilla_readability_and____github_com___348__-_peterdehaan_gmail_com_-_gmail_and_el_presidente_gal___realandreasgal____twitter_and_issues_ _mozilla_readability_ _github](https://cloud.githubusercontent.com/assets/557895/7150262/62817d80-e2cd-11e4-9a6b-b54e611c42bf.png)

**Comment by pdehaan** [comment]
Actually, to clarify, I don't think the _generic_ https://twitter.com/RealAndreasGal/ page should be reader-able, but I'd expect _specific_ Twitter tweets (ie: [twitter.com/RealAndreasGal/status/588582184258142208]( https://twitter.com/RealAndreasGal/status/588582184258142208)) to be available in reader view since they are static (although obviously not much content).

**Comment by pdehaan** [comment]
Not sure how/if this would work with tweets that have lots of replies, such as https://twitter.com/elonmusk/status/588144086755999744

I think a lot of these questions come back to the larger question of having lots of site-specific logic/CSS in readability for a lot of the major sites.

**Comment by gijsk** [comment]
Yeah, I really don't know how to make readability and/or isProbablyReaderable figure out what to do here. From a DOM structure perspective, there is almost no difference between the two "types" of tweet pages you listed. Suggestions?

**Comment by pdehaan** [comment]
I only have one idea, but it/me is stupid:

Profile page [1]: https://twitter.com/elonmusk
Tweet page [2]: https://twitter.com/elonmusk/status/588144086755999744

I noticed that the profile page [1] doesn't seem to have any open graph metadata tags:
```
$('meta')
[
  <meta charset=​"utf-8">​,
  <meta name=​"msapplication-TileImage" content=​"/​/​abs.twimg.com/​favicons/​win8-tile-144.png">​,
  <meta name=​"msapplication-TileColor" content=​"#00aced">​,
  <meta name=​"swift-page-name" id=​"swift-page-name" content=​"profile">
​]
```

But if I go to a tweets/details page [2], I do get some more open graph metadata tags:
```
$('meta')
[
  <meta charset=​"utf-8">​,
  <meta name=​"msapplication-TileImage" content=​"/​/​abs.twimg.com/​favicons/​win8-tile-144.png">​,
  <meta name=​"msapplication-TileColor" content=​"#00aced">​,
  <meta name=​"swift-page-name" id=​"swift-page-name" content=​"permalink">​,

  <meta property=​"og:​type" content=​"article">​,
  <meta property=​"og:​url" content=​"https:​/​/​twitter.com/​elonmusk/​status/​588144086755999744">​,
  <meta property=​"og:​title" content=​"Elon Musk on Twitter">​,
  <meta property=​"og:​image" content=​"https:​/​/​pbs.twimg.com/​profile_images/​579207936620138496/​o1LpNXMK_400x400.jpg">​,
  <meta property=​"og:​description" content=​"“If this works, I'm treating myself to a volcano lair. It's time.”">​, 
  <meta property=​"og:​site_name" content=​"Twitter">​,
  <meta property=​"fb:​app_id" content=​"2231777543">​
]
```

But it is probably a bit gross to have some giant "if (twitter) { do X }" block.
```js
console.dir($("meta[property='og:type']")[0].content) // "article"
```
